code/main/core/identityModule.brs

```diff
@@ -89,7 +89,7 @@ function _adb_IdentityModule(configurationModule as object) as object
                 return invalid
             end if
 
-            url = _adb_buildEdgeRequestURL(configId, _adb_generate_UUID(), m._EDGE_REQUEST_PATH, edgeDomain)
+            url = _adb_buildEdgeRequestURL(configId, _adb_generate_UUID(), m._EDGE_REQUEST_PATH, invalid, edgeDomain)
```

Comment:
nit: 
You don't have to do it now but in the future, if you add any more params, I recommend switching to either a builder pattern or modifying the method to accept an object, making it more manageable.

params = {
    configId: "yourConfigId",
    requestId: "yourRequestId",
    path: "yourPath",
    locationHint: yourLocationHint,
    edgeDomain: yourEdgeDomain
}
url = _adb_buildEdgeRequestURL(params)

Reply:
Yes, that sounds good.